charts/prometheus-blackbox-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 4.6.0
+version: 4.6.1
```

Comment:
Could you bump minor version instead of patch? This is new feature according to semver.